utils/validate.helpers.js

```diff
@@ -64,45 +64,16 @@ const checkErrors = template => {
 
     saveSessionData(req)
 
+    // flash error messages and redirect back on error
     if (!errors.isEmpty()) {
-      return renderPageWithErrors(req, res, {
-        template,
-        errors: errorArray2ErrorObject(errors),
-      })
+      req.session.flashmessage = errorArray2ErrorObject(errors)
+      return res.redirect('back')
```

Comment:
Where is the code that checks this flash message? I think we need to load it into `res.locals.errors` for the form-control errors to work.